packages/@aws-cdk/aws-cloudformation-codepipeline/README.md

```diff
@@ -0,0 +1,50 @@
+## AWS CodePipline Actions for AWS CloudFormation
+
+This module contains an Action that allows you to use CloudFormation actions from CodePipeline.
+
+Example usage:
+
+```ts
+import { Repository } from '@aws-cdk/aws-codecommit';
+import { PipelineSource } from '@aws-cdk/aws-codecommit-codepipeline';
+import { ArtifactPath, Pipeline, Stage } from '@aws-cdk/aws-codepipeline';
+import { Role } from '@aws-cdk/aws-iam';
+import { PolicyStatement, ServicePrincipal } from '@aws-cdk/cdk';
+import { CreateReplaceChangeSet, ExecuteChangeSet } from '../lib/pipeline-action';
```

Comment:
- This is not how users would use this module.
- I don't think there's a need to explicitly call out all the imports here.
- Also, we have been standardizing around namespaced imports: `import cdk = require('@aws-cdk/cdk')`, etc.